content/2017.04.getting-started-with-elm.rst

```diff
@@ -136,14 +149,14 @@ will be installed:
 
     Packages configured successfully!
 
-You now have a ``elm-package.json`` in your project as well as a
+You now have an ``elm-package.json`` file in your project as well as a
 ``elm-stuff/`` directory that contains libraries that ``elm-package``
-installed. ``elm-package.json`` is to Elm_ projects what
+installed. ``elm-package.json`` is to Elm projects what
 ``package.json`` is to nodejs ones.
 
 .. code-block:: bash
 
-	.
+    ~/tutorial$ tree -L2
```

Comment:
I know, still it was confusing. I'm finding this version more legible. I can revert this change if you heavily insist though.